lib/rubygems/defaults/ruby.rb

```diff
@@ -0,0 +1,37 @@
+module Gem
+  class << self
+
+    def bundled_gems_dir
+      File.join(RbConfig::CONFIG["rubylibprefix"], "bundled_gems", RbConfig::CONFIG["ruby_version"])
+    end
+
+    def default_gems_dir
+      File.join(RbConfig::CONFIG["rubylibprefix"], "default_gems", RbConfig::CONFIG["ruby_version"])
+    end
+
+    undef :default_specifications_dir if method_defined? :default_specifications_dir
+
+    if method_defined? :default_path
+      alias orig_default_path default_path
+      undef :default_path
+    end
+
+    ##
+    # Path to specification files of default gems.
+
+    def default_specifications_dir
+      @default_specifications_dir ||= File.join(Gem.default_gems_dir, "specifications", "default")
```

Comment:
Why is this cached while `bundled_gems_dir` and `default_gems_dir` aren't?

Reply:
This is cached also by RubyGems:

https://github.com/ruby/ruby/blame/209a0253f5869678d7228731605a1a5f21c76f32/lib/rubygems/defaults.rb#L63

and some parts of the test suite overrides the cached value (if I am not mistaken).

Also, I don't expect that `bundled_gems_dir` and `default_gems_dir` would be used by anything else then just by the `default_specifications_dir` and the `default_path` method, where the latter is cached here:

https://github.com/ruby/ruby/blob/209a0253f5869678d7228731605a1a5f21c76f32/lib/rubygems.rb#L348

Therefore the need for caching is minimal IMHO